fixtures/dev-registry/tests/dev-registry.test.ts

```diff
@@ -44,9 +44,29 @@ async function runWranglerDev(
 		{ WRANGLER_REGISTRY_PATH: devRegistryPath }
 	);
 
+	onTestFinished(() => session.stop());
+
 	return `http://${session.ip}:${session.port}`;
 }
 
+async function setupPlatformProxy(config: string, devRegistryPath?: string) {
+	vi.stubEnv("WRANGLER_REGISTRY_PATH", devRegistryPath);
+	try {
+		// @ts-expect-error It works
+		const wrangler = await import("wrangler");
+		const proxy = await wrangler.getPlatformProxy<Record<string, any>>({
+			configPath: config,
+		});
+
+		onTestFinished(() => proxy.dispose());
+
+		return proxy;
+	} finally {
+		// Unset the environment variable to avoid affecting other tests
+		vi.stubEnv("WRANGLER_REGISTRY_PATH", undefined);
```

Comment:
Shouldn't stub env automatically revert the change?

Reply:
Or if not, perhaps call `vi.unstubAllEnvs()` in the `onTestFinished()` hook

Reply:
Called vi.unstubAllEnvs() in the onTestFinished() hook in https://github.com/cloudflare/workers-sdk/pull/10060/commits/88c583bfcebe20911bf4bfbc64a200da686e5ac9